lib/pliny/error_reporter.rb

```diff
@@ -14,29 +13,35 @@ def self.notify(exception, context: {}, rack_env: {})
     end
   end
 
-  class RollbarReporter
-    include ::Rollbar::ExceptionReporter
-    include ::Rollbar::RequestDataExtractor
+  begin
```

Comment:
Maybe? I didn't try even one other approach, I mostly wanted to make sure that this was the only problem with rollbar being optional. So far, it seems to be the case.

Maybe it could live in a templated initializer?